x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -390,6 +415,21 @@ static RoleDescriptor kibanaSystem(String name) {
                         TransportUpdateSettingsAction.TYPE.name()
                     )
                     .build(),
+                RoleDescriptor.IndicesPrivileges.builder()
+                    .indices("logs-wiz.vulnerability-*")
+                    .privileges("read", "view_index_metadata")
+                    .build(),
```

Comment:
Looks good - since `logs-*` is a known collision pattern https://www.elastic.co/guide/en/elasticsearch/reference/current/index-templates.html